Docs 6444 3.9 connector docs (#971)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/microsoft-dynamics-gp-connector.adoc

= Microsoft Dynamics GP Connector - Mule 3
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

> Anypoint Connector for Microsoft Dynamics GP enables Mule apps to interact with the Microsoft Dynamics Great Plains (GP) Web Services.

Reviewer: ```suggestion
Anypoint Connector for Microsoft Dynamics GP (Microsoft Dynamics GP Connector) enables Mule apps to interact with the Microsoft Dynamics Great Plains (GP) Web Services.
```

> Read through this user guide to understand how to set up and configure a basic flow using the connector. 

Reviewer: ```suggestion
Read the provided user guide to understand how to set up and configure a basic flow using the connector. 
```

Release Notes: xref:release-notes::connector/microsoft-dynamics-gp-connector-release-notes.adoc[Microsoft Dynamics GP Connector Release Notes] +

> Reference: https://mulesoft.github.io/mule-microsoft-dynamics-gp-connector/[Mule Microsoft Dynamics GP Connector Reference].

Reviewer: Make these sentences.

== Prerequisites

> This document assumes that you are familiar with Microsoft Dynamics GP, Mule, Anypoint Connectors,

Reviewer: GLOBAL EDIT FOR TEMPLATE
```suggestion
You are familiar with Microsoft Dynamics GP, Mule runtime engine, Anypoint Connectors,
```

> Anypoint Studio essentials, elements in a Mule flow, and global elements.

Reviewer: In previous versions of this text, Global Elements is init. capped.

Reviewer: ***HERE***
